cesiumOmniverseCppTestsExtension/public/UsdUtilTests.cpp

```diff
@@ -0,0 +1,20 @@
+// #define DOCTEST_CONFIG_IMPLEMENT
+// #define DOCTEST_CONFIG_IMPLEMENTATION_IN_DLL
+// #define DOCTEST_CONFIG_SUPER_FAST_ASSERTS
+
+#include "UsdUtilTests.h"
+
+#include "cesium/omniverse/Context.h"
+#include "cesium/omniverse/LoggerSink.h"
+#include "cesium/omniverse/UsdUtil.h"
+
+#include <doctest/doctest.h>
+
+#include <iostream>
+
+void run_all_UsdUtil_tests() {
+    using namespace cesium::omniverse;
+    CESIUM_LOG_INFO("Running UsdUtil Tests...");
+    CHECK(cesium::omniverse::UsdUtil::primExists(pxr::SdfPath("/Cesium")));
```

Comment:
You can drop the `cesium::omniverse::` since there's the `using namespace cesium::omniverse` above

```suggestion
    using namespace cesium::omniverse;
    CESIUM_LOG_INFO("Running UsdUtil Tests...");
    CHECK(UsdUtil::primExists(pxr::SdfPath("/Cesium")));
```

or remove `using namespace cesium::omniverse`

```suggestion
    CESIUM_LOG_INFO("Running UsdUtil Tests...");
    CHECK(cesium::omniverse::UsdUtil::primExists(pxr::SdfPath("/Cesium")));
```

Reply:
`CESIUM_LOG_INFO` requires the cesium::omniverse namespace so I just kept the `using` statement, thanks for catching the redundancy though